### ISSUE TITLE
NullPointerException, invoke SmoochService.a(java.lang.String, java.lang.String, io.smooch.core.SmoochCallback, boolean) on a null object

### ISSUE THREAD

**Issue by islamassi** [smooch/smooch-android#119]
**Bug Report**

App crashes with the following exception:

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void io.smooch.core.service.SmoochService.a(java.lang.String, java.lang.String, io.smooch.core.SmoochCallback, boolean)' on a null object reference
       at io.smooch.core.c$3$1.run(Unknown Source)
       at io.smooch.core.service.SmoochService.U(Unknown Source)
       at io.smooch.core.service.SmoochService.a(Unknown Source)
       at io.smooch.core.service.SmoochService.a(Unknown Source)
       at io.smooch.core.service.SmoochService$12.run(Unknown Source)
       at io.smooch.core.service.SmoochService$8$1.a(Unknown Source)
       at io.smooch.core.service.SmoochService$8$1.a(Unknown Source)
       at io.smooch.core.b.f$15.a(Unknown Source)
       at io.smooch.core.b.a.b$1$2.run(Unknown Source)
       at android.os.Handler.handleCallback(Handler.java:751)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6776)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1496)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1386)
```

    private synchronized void U() {
        Iterator var1 = this.m.iterator();

        while(var1.hasNext()) {
            Runnable var2 = (Runnable)var1.next();
            var2.run();
        }

        this.m.clear();
    }

Fabric:  http://crashes.to/s/35df14eea23

- SDK version: 5.12.5

**Comment by alavers** [comment]
Hi @islamassi, are you still able reproduce this issue with the latest release? https://github.com/smooch/smooch-android/releases/tag/5.13.0

**Comment by islamassi** [comment]
Hi @alavers ,

Sorry for my late reply. I upgraded to the latest release `5.13.2`. The issue is still appearing. 
http://crashes.to/s/35df14eea23

**Comment by alavers** [comment]
That crashlytics link is from June 10th. Are you sure those stack traces are being generated by apps running version `5.13.2`?

**Comment by islamassi** [comment]
@alavers 
Some of the crashes were reported before upgrading to `5.13.2` and some others reported after upgrading to `5.13.2`. The crash is still not solved:
http://crashes.to/s/985810f698a

**Comment by alavers** [comment]
Thanks, we'll look into it

**Comment by wmora** [comment]
Fixed in version [5.13.3](https://github.com/smooch/smooch-android/releases/tag/5.13.3). Let us know if the issue persists

**Comment by islamassi** [comment]
@wmora 
The issue still persists in version `5.14.1`.
Fabric: [http://crashes.to/s/2284b77106c](url)
